sphinx/source/conf.py

```diff
@@ -115,8 +114,24 @@
     'VERSION': version,
 }
 
-html_theme = 'bokeh'
+html_theme ='pydata_sphinx_theme'
+
+html_theme_options = {
+    'external_links': [
+        {'name': 'Tutorial',  'url': 'https://mybinder.org/v2/gh/bokeh/bokeh-notebooks/master?filepath=tutorial%2F00%20-%20Introduction%20and%20Setup.ipynb'},
+        {'name': 'Community', 'url': 'https://discourse.bokeh.org'}
+    ],
+    'github_url': 'https://github.com/bokeh/bokeh',
+    'google_analytics_id': 'UA-27761864-7',
+#    'logo_link': 'https://static.bokeh.org/branding/logos/bokeh-logo.svg',
+    'show_toc_level': 2,
+    'twitter_url': 'https://twitter.com/bokeh',
+}
 
-html_theme_path = ['.']
+html_sidebars = {
+  "gallery": [],
+  "gallery/*": [],
+  "index": [],
```

Comment:
This setting not seem to have the desired effect (or any effect?)

Reply:
This setting requires a path (`"docs/gallery"`), not just a file name.